test/spec/windshaft/map-serializer/anonymous-map-serializer/anonymous-map-serializer.spec.js

```diff
@@ -0,0 +1,112 @@
+var Backbone = require('backbone');
```

Comment:
@iago I added this very basic tests to ensure anonymous-map-serializer is doing the right thing.